### ISSUE TITLE
Using DomParser

### ISSUE THREAD

**Issue by oliverjam** [FAC-Sixteen/b-b#4]
So I _think_ I know what you're using these for—the questions have HTML entities that you want to turn back into regular characters? A slightly easier way would be to use React's built-in handling of HTML:

```jsx
<div dangerouslySetInnerHTML=${{ __html: "What is Doug Walker&#039;s YouTube name?" }} />
```
It's generally discouraged because setting innerHTML is [a bit unsafe](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#Security_considerations), but assuming you can trust the stuff this API returns it's fine.

**Comment by bantek89** [comment]
this didn't work :( have we used it correctly?

![image](https://user-images.githubusercontent.com/23580331/58702694-61c6ca00-839e-11e9-8b57-0a90d85b8fd1.png)

![image](https://user-images.githubusercontent.com/23580331/58702713-75723080-839e-11e9-8c34-17c106066551.png)

![image](https://user-images.githubusercontent.com/23580331/58702725-802cc580-839e-11e9-9a85-24f0f6cee62b.png)

**Comment by oliverjam** [comment]
Yeah sorry, I should have been clearer. `dangerouslySetInnerHTML` is a property you apply to the DOM element itself. So in this case the `h2`

```jsx
return <h2 dangerouslySetInnerHTML={{ __html: question }} />
```

It's the react equivalent of doing something like
```js
document.querySelector("h2").innerHTML = question;
```
